projects/components/src/table/table.component.ts

```diff
@@ -504,6 +508,18 @@ export class TableComponent
       .subscribe(sort => this.updateSort(sort));
   }
 
+  public onLayoutChange(): void {
+    const latestTableWidth = this.table.nativeElement.offsetWidth;
+
+    if (this.tableWidth === 0) {
```

Comment:
so, we are only overriding for the first time?

Reply:
Initially, we just assign the width the first time (when the page loads). After that when we change the nav then we distribute the width